packages/bot/src/components/flyout.jsx

```diff
@@ -16,7 +18,13 @@ const Flyout = ({
 }) => {
     return (
         <div
-            className={`flyout${!is_visible ? ' hidden' : ''}${is_help_content ? ' flyout__help' : ' flyout__content'}`}
+            className={
+                classNames(
```

Comment:
Please update this, the `classnames` package's goal is to simplify adding conditional classes, please use its functionality. You can check the docs here https://github.com/JedWatson/classnames